### PR TITLE
fix(dsci): missing label on namespaces

### PR DESCRIPTION
**Pull request by zdtsw**
according to https://github.com/red-hat-data-services/odh-deployer/blob/main/deploy.sh#L113
we need all 3 labels on monitoring, application 2 namespaces.

ref: https://github.com/opendatahub-io/opendatahub-operator/issues/357

related: https://issues.redhat.com/browse/RHODS-7978

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,16 +3,16 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-IMAGE_OWNER ?= opendatahub
-VERSION ?= 2.4.0
+IMAGE_OWNER ?= wenzhou
```
Comment (VaishnaviHire) [inline Makefile:6]: Probably need to revert this change?

File: bundle/manifests/rhods-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -121,7 +121,7 @@ metadata:
     operators.operatorframework.io/internal-objects: '[dscinitialization.opendatahub.io]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/red-hat-data-services/rhods-operator
-  name: rhods-operator.v2.4.0
+  name: rhods-operator.v2.10.30-0
```
Comment (VaishnaviHire) [inline bundle/manifests/rhods-operator.clusterserviceversion.yaml:124]: Same as above

File: bundle/manifests/rhods-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -1571,7 +1677,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: REPLACE_IMAGE:latest
+                image: quay.io/wenzhou/opendatahub-operator:dev-2.10.30-0
```
Comment (VaishnaviHire) [inline bundle/manifests/rhods-operator.clusterserviceversion.yaml:1680]: Same as above

File: controllers/dscinitialization/utils.go [hunk 4]
```diff
@@ -101,6 +103,14 @@ func (r *DSCInitializationReconciler) createOdhNamespace(ctx context.Context, ds
 				r.Log.Error(err, "Unable to fetch monitoring namespace", "name", monitoringName)
 				return err
 			}
+		} else { // force to patch monitoring namespace with label for cluster-monitoring
+			r.Log.Info("Patching monitoring namespace for Managed cluster", "name", monitoringName)
```
Comment (VaishnaviHire) [inline controllers/dscinitialization/utils.go:107]: Wouldn't this add the namespace label https://github.com/red-hat-data-services/rhods-operator/pull/98/files#diff-5f4497af58ef4b32b811ace8e826c31377aa6ddd140b859e88dd84b876e7139aR93?

File: Makefile (hunk 1 above)
Comment (zdtsw) [reply Makefile:6]: yeah i will need to cleanup code first
